### PR TITLE
Replace Nats MQ with SQS in microservices

### LINKED ISSUE #517: Replace Nats with SQS in queuer and queue consumer services
- Replace publishing of messages to Nats server with amazon SQS in the Queuer service.
- Replace reading messages from Nats to SQS in the queue consumer.

### PR DESCRIPTION
**Pull request by FyzHsn**
Fixes #517

### CODE REVIEW

**Comment by FyzHsn** [comment]
@AlejandroEsquivel I ended up taking out the try / except block for the msg_body. When the int / string msgs were being published to the queue and received in the queue_polling method, it wasn't raising any errors. The error was being raised in the msg handler.

**Review by scottwn — APPROVED** [review]

**Review by AlejandroEsquivel — CHANGES_REQUESTED** [review]

File: covalent_queuer/app/api/api_v0/endpoints/submit.py [hunk 1]
```diff
@@ -19,15 +19,18 @@
...
 import logging
+import os
 import uuid
 from io import BytesIO
 
 import cloudpickle as pickle
 from app.core.api import ResultsService
-from app.core.queuer import Queuer
-from app.schemas.submit import ResultPickle, SubmitResponse
+from app.core.queuer import Queue
```
Comment (AlejandroEsquivel) [inline covalent_queuer/app/api/api_v0/endpoints/submit.py:28]: `app.core.queuer` should be `app.core.queue`

**Review by AlejandroEsquivel — CHANGES_REQUESTED** [review]

File: covalent_queuer/tests/app/core/queuer_test.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from app.core.queuer import Queue
+
+
+class TestQueuer:
+
+    MOCK_MQ_CONNECTION_URI = "localhost:4222"
+
+    @mock.patch.object(nats, "connect", autospec=True)
+    @mock.patch.dict(os.environ, {"MQ_CONNECTION_URI": MOCK_MQ_CONNECTION_URI})
+    def test_get_client(self, mocked_nats):
```
Comment (AlejandroEsquivel) [inline covalent_queuer/tests/app/core/queuer_test.py:15]: Might need to be skipped for now until test is replaced

Comment (FyzHsn) [reply covalent_queuer/tests/app/core/queuer_test.py:15]: Done.

**Review by AlejandroEsquivel — APPROVED** [review]

**Review by scottwn — APPROVED** [review]

**Pull request merged**